pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -67,6 +67,9 @@ func (*OSSMigrations) AddMigration(mg *Migrator) {
 	addKVStoreMigrations(mg)
 	ualert.AddDashboardUIDPanelIDMigration(mg)
 	accesscontrol.AddMigration(mg)
+	if mg.Cfg != nil && mg.Cfg.FeatureToggles["accesscontrol"] {
+		accesscontrol.AddTeamMembershipMigrations(mg)
+	}
```

Comment:
this is not something that's need to change. However our database implementation is now "stable" and we have the access control out for customers so we do not have to check the feature toggle to do more migrations further. qouting Emil a bit here

just for next time with running migrations 👍